plugin/Renderer/InputFieldItalicLabelCheckboxRenderer.php

```diff
@@ -0,0 +1,69 @@
+<?php
```

Comment:
From the code I'm not sure I understand. What problem does this class solve?

Reply:
This class is for this kind of label which mix normal font with italic font
![image](https://user-images.githubusercontent.com/72601287/168511725-3dcde27b-4c10-4710-aa5f-e6109ba72425.png)

Reply:
Why do we need a separate class for this? Is it because the other class only takes text and this needs HTML?

Reply:
You're right

Reply:
You're right